src/Frontend/Components/AttributionCountPerSourcePerLicenseTable/AttributionCountPerSourcePerLicenseTableFooter/AttributionCountPerSourcePerLicenseTableFooter.tsx

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: Meta Platforms, Inc. and its affiliates
+// SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+//
+// SPDX-License-Identifier: Apache-2.0
+import MuiTableCell from '@mui/material/TableCell';
+import MuiTableFooter from '@mui/material/TableFooter';
+import MuiTableRow from '@mui/material/TableRow';
+import _ from 'lodash';
```

Comment:
Other places in OpossumUI import the functions they need directly:
```suggestion
import { sum } from 'lodash';
```
But in this case, I think you don't even need lodash as the native solution isn't too verbose:
```
Object.values(props.totalAttributionsPerSource).reduce((acc, count) => acc + count, 0)
```